src/odemis/acq/stream/_sync.py

```diff
@@ -1301,13 +1301,20 @@ def _runAcquisitionEbeam(self, future):
                         # Live update the setting stream with the new data
                         try:
                             self._sccd._onNewData(self._ccd_df, self._acq_data[self._ccd_idx][-1])
+                            if integration_count > 10:
+                                logging.debug("Sum the acquired images one after another")
+                                self._integrateImages(integration_count)
```

Comment:
That shouldn't be inside this block. It's reserved for `_onNewData()` which is "allowed" to fail because it's just an extra feature.

Also drop this special case for 10. Always do the code path.